src/frontend/src/binder/insert.rs

```diff
@@ -164,6 +190,8 @@ impl Binder {
             source,
             cast_exprs,
             column_idxs: target_table_col_idxs,
+            returning_list,
+            schema: Schema { fields },
```

Comment:
This sounds a little confusing, as if no `RETURNING`, this will be empty as well, instead of the Int64 "affected rows". How about naming it `returning_schema`?